src/test/resources/testbib/bibWithEpilog.bib

```diff
@@ -0,0 +1,43 @@
+% Encoding: UTF-8
+
+@Preamble{preamble}
+
+@String{firstString  = {my first string}}
+@String{secondString = {}}
+
+@ARTICLE{1102917,
+  author = {E. Bardram},
+  title = {The trouble with login: on usability and computer security in ubiquitous
+	computing},
+  journal = {Personal Ubiquitous Comput.},
+  year = {2005},
+  volume = {9},
+  pages = {357--367},
+  number = {6},
+  address = {London, UK},
+  bdsk-url-1 = {http://dx.doi.org/10.1007/s00779-005-0347-6},
+  doi = {http://dx.doi.org/10.1007/s00779-005-0347-6},
+  issn = {1617-4909},
+  publisher = {Springer-Verlag}
+}
+
+@INPROCEEDINGS{1137631,
+  author = {Gustav Bostr\"{o}m and Jaana W\"{a}yrynen and Marine Bod\'{e}n and
+	Konstantin Beznosov and Philippe Kruchten},
+  title = {Extending XP practices to support security requirements engineering},
+  booktitle = {SESS '06: Proceedings of the 2006 international workshop on Software
+	engineering for secure systems},
+  year = {2006},
+  pages = {11--18},
+  address = {New York, NY, USA},
+  publisher = {ACM},
+  bdsk-url-1 = {http://doi.acm.org/10.1145/1137627.1137631},
+  doi = {http://doi.acm.org/10.1145/1137627.1137631},
+  file = {:/Volumes/iDisk/Freie Universität Berlin/Semester 9/Softwareprozesse/p11-bostrom.pdf:PDF},
+  isbn = {1-59593-411-1},
+  location = {Shanghai, China}
+}
+
+@Comment{jabref-meta: databaseType:bibtex;}
```

Comment:
I think you missunderstood me. For complex.bib there are already round-trip tests in place. Thus you just need to modify `complex.bib` instead of writing a new round-trip test with a new bib file.